src/components/DonateCard.tsx

```diff
@@ -216,6 +216,7 @@ const DonateCard = (props: DonateCardProps) => {
             />
             <Box center>
                 <DropdownMenu
+                    {...({} as any)}
```

Comment:
It reminds me of the issues I've had with the website when updating node and typescript. It's probably worth trying to understand the problem now. I can also try to run this locally and see how it goes.

Reply:
without this we get typescript errors in <ViewContainer> <DropdownMenu> and <Grid>. It's happening in main.
If i'm right those typescript errors are types that are missing in UI on those components. So, with those errors in webapp it never passed the tests on vercel which means it wont build the app.
This work around is basically to add all props to "any" type.

Reply:
For now and to not block this, its not a big deal but yeah whenever we have the chance, we should address the underlying issue because its a bit weird to be adding deconstruct of an empty object all over the place.